scripts/visualize.py

```diff
@@ -2,8 +2,8 @@
 
 import sys
 
-import ogr
 from liblas import file
```

Comment:
This thing should really be dropped or updated to use laspy for fetching information. It is a very old holdover from a long time ago.